uint8 digital_outputs
float64 voltage0
float64 voltage1
float64 timeout
